css/at-rules/container.json

```
@@ -36,6 +36,102 @@
             "deprecated": false
           }
         },
+        "anchor_position_queries": {
+          "__compat": {
+            "description": "Anchor position queries (`anchored()`)",
+            "spec_url": "https://drafts.csswg.org/css-anchor-position-2/#anchored-container-queries",
+            "support": {
+              "chrome": {
+                "version_added": "143"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "fallback": {
+            "__compat": {
+              "description": "`fallback` container feature",
+              "spec_url": "https://drafts.csswg.org/css-anchor-position-2/#anchored",
+              "support": {
+                "chrome": {
+                  "version_added": "143"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror",
+                "webview_ios": "mirror"
+              },
+              "status": {
+                "experimental": true,
+                "standard_track": true,
+                "deprecated": false
+              }
+            },
+            "any": {
+              "__compat": {
+                "description": "`fallback` container feature",
+                "spec_url": "https://drafts.csswg.org/css-anchor-position-2/#anchored",
```

Is this `position-area-query: any`?

```suggestion
            "any": {
              "__compat": {
                "description": "`any` container feature",
                "spec_url": "https://drafts.csswg.org/css-anchor-position-2/#valdef-position-area-query-any",
```